src/Compilers/CSharp/Portable/Symbols/Attributes/PEAttributeData.cs

```diff
@@ -171,6 +171,41 @@ internal override bool HasErrors
             }
         }
 
+        internal override DiagnosticInfo? ErrorInfo
+        {
+            get
+            {
+                if (HasErrors)
+                {
+                    switch (AttributeConstructor)
+                    {
+                        case { HasUseSiteError: true } attributeConstructor:
+                            return attributeConstructor.GetUseSiteInfo().DiagnosticInfo;
+
+                        case { }:
+                            return new CSDiagnosticInfo(ErrorCode.ERR_BogusType, string.Empty);
```

Comment:
>string.Empty [](http://example.com/codeflow?start=81&length=12)

`AttributeClass`?

Reply:
> ```AttributeClass```?

No. There is nothing wrong with the attribute class here. We simply failed to crack the attribute application, i.e. dealing with some sort of invalid metadata for the specific attribute application. In such cases elsewhere we create  ```UnsupportedMetadataTypeSymbol``` and this is how we report a use-site error for it.